kickstarter/android-oss
# NTV-208: Scrolling transition of the video player

# 📲 What

Create scrolling animation for media header. 

# 🤔 Why

Project page revamp.

# 🛠 How

Created a motion layout and motion scene that listens to tab clicks, and starts/stops the animation upon accordingly. 

# 👀 See

https://user-images.githubusercontent.com/19390326/140775585-94acaf21-6176-4c2e-ac31-5b46f7eb8efc.mp4

# 📋 QA

- Open a project
- Tap on a tab, media header should scroll out of view
- Tap on the overview tab, media header should animate into view
- In the overview tab, swipe up to scroll. Video should scroll inline, tool bar should be pinned to the top.

# Story 📖

[NTV-208: Scrolling transition of the video player](https://kickstarter.atlassian.net/browse/NTV-208)
